src/BasePlatform.js

```diff
@@ -162,4 +162,6 @@ export default class BasePlatform {
     getEventIndexingManager(): BaseEventIndexManager | null {
         return null;
     }
+
+    async setLanguage(preferredLangs: string[]): Promise<void> {}
```

Comment:
If nothing's actually going to await on the the result it might make more sense for this to just be a regular function at the superclass level (also don't forget this is still a flow-typed file, although I suspect your type definition is valid in both).